src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,164 @@ Project has the following structure:
+                description: One procedure has thrown an error or has not reported a status in time
+      ...
+      ```
+      Our Starter Kit template comes with example paths and methods defined. Let's see the `/healthcheck` API. It is used to express the current state of our application in very simple terms: UP or DOWN.
+      
+      We can see that it is registered under `/healthcheck` path, it's method is a `GET` and it can have various responses depending on the state of the instance.
+      
+      There's no magic behind it. Healthcheck is handled by operation (more on that later in this tutorial) `healthcheck-operation` implemented as a handler in `HealthcheckHandlerFactory.java`. We will take a look at it later in this tutorial. Just to be brief: it simply calls our example endpoints (defined in `knotx/conf/openapi.yaml` as well). 
+      
+      This is a nice feature to have! Since we will be using Docker in this tutorial, we will tell docker to check if our instance is up and running by simply calling this endpoint and expecting a response.
+      
+      Let's take a quick peek at `docker/Dockerfile`:
+      
+      ```dockerfile
+      ... 
+      
+      HEALTHCHECK --interval=5s --timeout=2s --retries=12 \
+        CMD curl --silent --fail localhost:8092/healthcheck || exit 1
+        
+      ...
+      ```
+      Quite self-explanatory, isn't it? For more details see [this documentation](https://docs.docker.com/engine/reference/builder/#healthcheck).
+      
+Now that, we know what's going on in this file, let's define our custom endpoint.
+
+Simply add new routing configuration under `paths`:
+```hocon
+paths:
+  /api/hello:
     get:
-      operationId: content-get
+      operationId: hello-world-operation
       responses:
         default:
-          description: Remote repository template processing
-``` 
+          description: Example API implemented in this tutorial
+```
 
-By doing this you define the operation which should be executed for path `/content/*`.
+Simple as that! 
 
-### operations.conf
+Now let's take a closer look at what lies under `operationId: hello-world-operation` ...
 
-Now, you need to define operation `content-get`. Open `knotx/conf/routes/operations.con`
-Add in `routingOperations` array following definition:
+## Create a new operation
 
+As you surely noticed, paths have something to do with [operations](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#operationObject). [Routing operations](https://github.com/Knotx/knotx-server-http/tree/master#routing-operations) are defined in `knotx/conf/routes/operations.conf`. 
+
+Let's take a look at this file:
+
+```hocon
+routingOperations = ${routingOperations} [
+  
+  // routing operations for this project
+  
+]
 ```
+
+`routingOperations` is an array of `routingOperation` objects. 
+
+`${routingOperations}` takes care of merging any previously defined routing operations with the ones you define here.
```

Comment:
Good place to link to the appropriate place in HOCON specification

Reply:
done